app/components/candidate_interface/continuous_applications/application_review_and_submit_component.html.erb

```diff
@@ -1,15 +1,4 @@
-<% if application_can_submit? %>
-  <h2 class="govuk-heading-m">Review and submit your application</h2>
-  <p class="govuk-body">
-    You will have the opportunity to review your application in full before you submit it.
-  </p>
-
-  <div class="govuk-button-group app-course-choice__confirm-submission">
-    <%= govuk_button_to('Review application', review_path, method: :get) %>
-
-    <%= govuk_link_to('Save as draft', candidate_interface_continuous_applications_choices_path) %>
-  </div>
-<% else %>
+<% if !application_can_submit? %>
```

Comment:
Can we change to unless instead of the if not?

Reply:
Done